pages/userSettings/userSettings.ejs

```diff
@@ -2,6 +2,8 @@
 <html>
   <head>
     <%- include('../partials/head'); %>
+    <link href="https://cdn.jsdelivr.net/gh/gitbrent/bootstrap4-toggle@3.6.1/css/bootstrap4-toggle.min.css" rel="stylesheet">
+    <script src="https://cdn.jsdelivr.net/gh/gitbrent/bootstrap4-toggle@3.6.1/js/bootstrap4-toggle.min.js"></script>
```

Comment:
We can't include anything from a CDN because PL needs to work without a network.